unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
We don't have anything in place to deal with qstrs needed by an external module like lwIP.

For example, extmod/modubinascii needs some qstr's like hexlify, which currently comes from here: https://github.com/micropython/micropython/blob/master/py/qstrdefs.h#L597-L603

I took a look at the stmhal Makefile, and there are some other places we get qstr's from:
https://github.com/micropython/micropython/blob/master/stmhal/Makefile#L16

When we process the pins file for a particular board we generate all of the qstr's needed to define the pins, and similarly, for the stmmodule we actually parse some C header files to extract offsets and base addresses and we also generate qstr's that are needed for that.

I think that in an ideal world, that lwIP should have a qstrdefs.h file that describes all of the qstrs that it needs, and that the port that wants lwIp should then add the appropriate qstrdefs.h file to the list of files to be processed.